main.js

```diff
@@ -129,4 +129,30 @@ const checkLowerCase = (input, event) => {
 getForm.addEventListener('submit', (e) => {
   e.preventDefault();
   checkLowerCase(getEmail);
-});
\ No newline at end of file
+});
+
+// Start LocalStorage
+
+const getName = document.querySelector('input[type="text"]');
+const getTextArea = document.querySelector('textarea');
+
+const dataStore = JSON.parse(localStorage.getItem('dataStore')) || [];
```

Comment:
- [ ] As it is stated in the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/10_localstorage.md) we should only store a single javascript object to store the form data.

**Requirement**
![Screenshot from 2023-01-07 14-09-18](https://user-images.githubusercontent.com/17243325/211147229-fa8108bf-96a0-433c-9056-04817dbb45af.png)

Every time a form input is updated it is creating an object and stores it in an array of objects. Please check the video below.

https://www.loom.com/share/0e41b7a9725c4264b4566dd4ddf7f122

Kindly update it to use a single object to store the data. ✍️